tests/Resources/Fixtures/config/full.xml

```diff
@@ -54,7 +54,7 @@
                 <my_service type="service" service="my_auth_service"/>
             </authentication>
             <cache cache-pool="my_cache_pool" stream-factory="my_other_stream_factory">
-                <config default-ttl="42" cache-lifetime="2592000" hash-algo="sha1" cache-key-generator="null">
+                <config default-ttl="42" cache-lifetime="2592000" hash-algo="sha1" cache-key-generator="null" cache-listeners="">
```

Comment:
that won't work. cache-listener needs to be a xml list, as in 
```
<config ...>
    <cache-listener>service-name</cache-listener>
    <cache-listener>other-service-name</cache-listener>
    ....
</config>
```

i'd prefer to have the test with 2 listeners so that we see it works as expected.

Reply:
ack